### PR TITLE
long footer fixed

### LINKED ISSUE #156: Long footer overlaps Table of Contents
image

### PR DESCRIPTION
**Pull request by mansoorbarri**
fixes #156 

long footer would wrap instead of going over TOC

image

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -290,6 +291,7 @@ a:hover .feather-sun {
   display: block;
   padding: 6px 12px;
   text-decoration: none;
+
```
Comment (yashmehrotra) [inline assets/css/main.css:294]: remove this empty line

**Comment by mansoorbarri** [comment]
it doesn't show the sapce on my fork

File: assets/css/main.css (hunk 1 above)
Comment (yashmehrotra) [reply assets/css/main.css:294]: In your code what does this line show then ?

Comment (mansoorbarri) [reply assets/css/main.css:294]:
```css
.post-tag a {
  color: inherit;
  display: block;
  padding: 6px 12px;
  text-decoration: none;
}
```

**Comment by mansoorbarri** [comment]
I cloned it again and checked, should be fine now

**Review by 526avijitgupta — CHANGES_REQUESTED** [review]
Hi @mansoorbarri thanks for fixing this.

One quick suggestion, we can make the footer a bit wider while still leaving some gap with the TOC sidebar. 50% width looks good to me, which means the margin-left needs to be adjusted to 25% too.

Can you please make the changes?

**Comment by mansoorbarri** [comment]
@526avijitgupta increasing the width makes the footer goes over the TOC: 
image

**Comment by 526avijitgupta** [comment]
@mansoorbarri Are you trying with the width set to 50%?

**Comment by mansoorbarri** [comment]
@526avijitgupta yes

**Comment by 526avijitgupta** [comment]
@mansoorbarri alright then I'll merge this PR for now. We can try to increase the width separately later

**Review by 526avijitgupta — APPROVED** [review]

**Pull request merged**